sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -186,6 +187,27 @@ const result = await client.createCertificate(
 Calling to `createCertificate` with the same name will create a new version of
 the same certificate, which will have the latest provided attributes.
 
+### Waiting until the certificate is created
+
+Certificates take some time to get fully created since they need to be signed
+by their respective Certificate Authority. Since this is a process that can
+take an unbounded amount of time, we're providing a special method
+`beginCreateCertificate`, which returns a Poller, which will be able to manage
+the underlying Long Running Operation and wait until the certificate finished
+being created. Here's a quick example on how to use this method:
+
+```typescript
+const certificateName = "MyCertificateName";
+const certificatePolicy = {
+  issuerName: "Self"
+};
+
+const poller = await client.beginCreateCertificate(certificateName, certificatePolicy);
+
+const result = await poller.done();
+console.log(result!.status); // "completed"
```

Comment:
Feels weird that result.status can be undefined... shouldn't it always be something?

Reply:
it's not that status can be undefined, it's that result can be undefined, because of the signature of `done()`. Should I throw if there's no result?

Reply:
ah, I feel like yeah, we should make done() have a definite result shape and reject the done promise when getResult is undefined

Reply:
I'll update it